src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource/help-repositoryUrl.html

```diff
@@ -0,0 +1,13 @@
+<div>
+    <p>Specify the HTTPS URL of the <strong>GitHub Organization/User</strong> and <strong>repository</strong>.</p>
+    <p> Github examples: </p>
+    <p>   https://github.com/jenkinsci/github-branch-source-plugin </p>
+    <p>   https://github.com/jenkinsci/github-branch-source-plugin.git </p>
```

Comment:
I don't know if we want to advertise this format. It seems fine to handle it to help users, but they might get the wrong idea and assume we support other remote formats as well.

Reply:
Required by PM